adal/src/androidTest/java/com/microsoft/aad/adal/StringExtensionTests.java

```diff
@@ -45,62 +45,44 @@ protected void tearDown() throws Exception {
     public void testIsNullOrBlankNotEmpty() throws IllegalArgumentException,
             IllegalAccessException, InvocationTargetException, ClassNotFoundException,
             NoSuchMethodException, InstantiationException {
-        final String methodName = "IsNullOrBlank";
-        Object foo = ReflectionUtils.getNonPublicInstance("com.microsoft.aad.adal.StringExtensions");
-        Method m = ReflectionUtils.getTestMethod(foo, methodName, String.class);
-        boolean result = (Boolean) m.invoke(foo, "non-empty");
-        assertFalse("not empty", result);
+        assertFalse("not empty", StringExtensions.isNullOrBlank("non-Empty"));
     }
 
     public void testIsNullOrBlankEmpty() throws IllegalArgumentException, IllegalAccessException,
             InvocationTargetException, ClassNotFoundException, NoSuchMethodException,
             InstantiationException {
-        final String methodName = "IsNullOrBlank";
-        Object foo = ReflectionUtils.getNonPublicInstance("com.microsoft.aad.adal.StringExtensions");
+        assertTrue("empty", StringExtensions.isNullOrBlank(""));
 
-        Method m = ReflectionUtils.getTestMethod(foo, methodName, String.class);
-        boolean result = (Boolean) m.invoke(foo, "");
-        assertTrue("empty", result);
+        assertTrue("empty", StringExtensions.isNullOrBlank("  "));
```

Comment:
For line 56 and lien 58, what is the difference between the two? It's all testing the trim, no need to have two.